### PR TITLE
Add Grantlee5 requirement to CMakeLists and convert Int to Long.

### PR DESCRIPTION
**Pull request by renlinch**
I had some fail on compile time. And could not create any Images with over 2 GiB Size.

### CODE REVIEW

File: src/lib/storagepool.h [hunk 1]
```diff
@@ -66,7 +66,7 @@ class StoragePool : public QObject
 
     bool build(int flags);
     bool create(int flags);
-    StorageVol *createStorageVolume(const QString &name, const QString &format, int sizeGiB, int flags);
+    StorageVol* createStorageVolume(const QString& name, const QString& format, long sizeGiB, int flags);
```
Comment (dantti) [inline src/lib/storagepool.h]: Please don't change the style

File: src/lib/storagepool.cpp [hunk 2]
```diff
@@ -174,27 +174,29 @@ bool StoragePool::create(int flags)
...
+StorageVol *StoragePool::createStorageVolume(const QString &name, const QString &format, long sizeGiB, int flags)
 {
     QByteArray output;
     QXmlStreamWriter stream(&output);
 
     QString localName(name);
     QString localFormat(format);
-    int sizeInt = sizeGiB * 1073741824;
-    int alloc = sizeInt;
+    long sizeInt = sizeGiB * 1073741824;
```
Comment (dantti) [inline src/lib/storagepool.cpp]: Use qint64, and QString::toLongLong()

File: src/storages.cpp [hunk 3]
```diff
@@ -134,7 +134,7 @@ void Storages::storage(Context *c, const QString &hostId, const QString &pool)
             if (params.contains(QStringLiteral("meta_prealloc")) && format == QLatin1String("qcow2")) {
                 flags = VIR_STORAGE_VOL_CREATE_PREALLOC_METADATA;
             }
-            StorageVol *vol = storage->createStorageVolume(name, format, size.toInt(), flags);
+            StorageVol *vol = storage->createStorageVolume(name, format, size.toLong(), flags);
```
Comment (dantti) [inline src/storages.cpp]: tolonglong()

File: src/lib/storagepool.cpp [hunk 4]
```diff
@@ -174,27 +174,29 @@ bool StoragePool::create(int flags)
...
+    QString size = QString::number(sizeInt);
+    QString alloc = size;
+    
     if (format == QLatin1String("unknown")) {
         localFormat = QStringLiteral("raw");
     }
     if (format == QLatin1String("dir")) {
         localName.append(QLatin1String(".img"));
-        alloc = 0;
+        alloc = "";
```
Comment (dantti) [inline src/lib/storagepool.cpp]: use QString(), I probably forgot to disable casting from ASCII

**Review by dantti — COMMENTED** [review]
please fix pending issues

**Comment by renlinch** [comment]
Sorry was my first C++ commit... and KDevelop did some automatic formatting ...

**Comment by dantti** [comment]
No problem, thanks.

**Pull request merged**